### ISSUE TITLE
Experimenting with Express and Libvirt-node keeps crashing

### ISSUE THREAD

**Issue by atwright147** [RamyElkest/libvirt-node#34]
Hey,

I am very excited about this project and have been experimenting. I built a project in Express.js to use take advantage of libvirt-node.

I have successfully got it going, but only for one request. After that one request it crashes with the following error:

```sh
node: ../generated/libvirt-connect.c:943: libvirt_virConnectListAllDomains: Assertion `status == napi_ok' failed.
```

Do you have any idea what might be wrong?

This is the current Express App code: https://gitlab.com/atwright147/kvm-controller

**Comment by RamyElkest** [comment]
seems virConnectListAllDomains returns -1
https://libvirt.org/html/libvirt-libvirt-domain.html#virConnectListAllDomains

```
        int c_retval = virConnectListAllDomains(arg0, &arg1, arg2);

        if(c_retval < 0) {
            status = napi_get_null(env, &n_retval);
            assert(status == napi_ok);
            return n_retval;
        }
```

**Comment by nertpinx** [comment]
Since this was fixed in #35 this can be closed, right?

**Comment by RamyElkest** [comment]
yup.
if you face any other issues please bring them forward @atwright147, your feedback is appreciated.
